README.md

```diff
@@ -96,9 +105,8 @@ Create a task manifest file (exemplary file in [examples/task/] (https://github.
                 "/intel/procfs/load/scheduling": {}
             },
             "config": {
-                "/intel/mock": {
-                    "password": "secret",
-                    "user": "root"
+                "/intel/procfs/load/": {
+                    "procfs_path": "/var/procfs/loadavg"
```

Comment:
For the CPU (https://github.com/intelsdi-x/snap-plugin-collector-cpu/pull/16) and Swap (https://github.com/intelsdi-x/snap-plugin-collector-swap/pull/8) plugins the option name is `proc_path`. I think the naming should be consistent.